docs/application/web/guides/security/privacy-related-permissions.md

```diff
@@ -19,7 +19,7 @@ The main features of the Privacy Privilege API include:
 For a list of privacy-related privileges, see [Security and API Privileges](../../tutorials/sec-privileges.md).
 
 > [!NOTE]
-> Since Tizen 5.5, if the caller app component type is UI app or component based app with frame component, then, the pop-ups by `requestPermission()` and `requestPermissions()` are launched as `group mode` with the caller apps.
+> Since Tizen 5.5, if the caller app component type is UI app then the pop-ups by `requestPermission()` and `requestPermissions()` are launched as `group mode` with the caller apps.
```

Comment:
How about below little changes?
```suggestion
> Since Tizen 5.5, if the caller application component type is UI application, then the pop-ups by `requestPermission()` and `requestPermissions()` are launched as `group mode` with the caller application.
```

Reply:
@mijin85cho yes looks fine.